packages/components/src/index.ts

```diff
@@ -131,6 +131,8 @@ export type {
 } from './molecules/ProductCard'
 export { default as ProductTitle } from './molecules/ProductTitle'
 export type { ProductTitleProps } from './molecules/ProductTitle'
+export { default as ProductPrice } from './molecules/ProductPrice'
+export type { ProductPriceProps } from './molecules/ProductPrice'
```

Comment:
Nit: can you place it before the `ProductTitle`? So that we try to keep it sorted alphabetically.